src/cli.ts

```diff
@@ -309,10 +309,14 @@ export async function run (process: NodeJS.Process): Promise<number> {
   }
 
   myConsole.log('LOG   | writing BOM to', options.outputFile)
+  const directory = dirname(options.outputFile)
+  if (!existsSync(directory)) {
+    mkdirSync(directory, { recursive: true })
+  }
   const written = await writeAllSync(
     options.outputFile === OutputStdOut
       ? process.stdout.fd
-      : openSync(resolve(process.cwd(), options.outputFile), 'w'),
```

Comment:
why would this change be needed?

Reply:
The `resolve(...paths: string[]): string;` method already uses the current working directory if no path is passed.

Reply:
so this line can be reverted, right?

Reply:
If the library only should support relative paths, then I will add the argument again.